apps/accounts/backends/__init__.py

```diff
@@ -13,9 +13,6 @@
 def get_vault_client(raise_exception=False, **kwargs):
     tp = kwargs.get('VAULT_BACKEND') if kwargs.get('VAULT_ENABLED') else VaultTypeChoices.local
 
-    # TODO: Temporary processing, subsequent deletion
-    tp = VaultTypeChoices.local if tp == VaultTypeChoices.azure else tp
-
     try:
         module_path = f'apps.accounts.backends.{tp}.main'
         client = import_module(module_path).Vault(**kwargs)
```

Comment:
There is no regularity to be checked in this code. It looks correct up until the import statement change. The current file does not have any errors or improvements suggested.